app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
Hi @Baayeh 
A great job is done so far

- [ ] I check your migration files and notice you added migrations for `author_id` on the different tables as requested by the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) but you did not reference the `author` in your model files.

### Please follow the sample code below and make the necessary correction to your different model files.
```ruby
# app/models/user.rb
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

# app/models/post.rb
belongs_to :author, class_name: 'User'

# app/models/comment.rb
belongs_to :author, class_name: 'User'

# app/models/like.rb
belongs_to :author, class_name: 'User'
```

Reply:
I have referenced the author in all the models as suggested